syncopy/shared/computational_routine.py

```diff
@@ -284,7 +281,7 @@ def initialize(self, data, out_stackingdim, chan_per_worker=None, keeptrials=Tru
         trials = []
         for tk, trialno in enumerate(self.trialList):
             trial = data._preview_trial(trialno)
-            trlArg = tuple(arg[tk] if isinstance(arg, Sized) and len(arg) == self.numTrials \
+            trlArg = tuple(arg[tk] if isinstance(arg, (list, tuple, np.ndarray)) and len(arg) == self.numTrials \
```

Comment:
This mechanism of (only!) distributing args (and not kwargs) along trials probably can still be made more robust and transparent.. but this we can/should tackle later

Reply:
Agreed - I think the current mechanism works well enough but it's desperately lacking documentation...